automatic/gpg4win/tools/chocolateyInstall.ps1

```diff
@@ -21,4 +21,4 @@ if ($pp['Config']) {
 }
 
 Install-ChocolateyInstallPackage @packageArgs
-Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { touch "$_.ignore" '' }}
+Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { New-Item "$($_.FullName ).ignore" }}
```

Comment:
Please change this line to:

```suggestion
Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { New-Item -ItemType File -Path "$($_).ignore" }}
```

This would make it consistent in the changes I made in the helper that expands aliases for the package scripts.